locale/tr/docs/guides/debugging-getting-started.md

```diff
@@ -0,0 +1,262 @@
+---
+title: hata ayıklama - Başlarken
+layout: docs.hbs
```

Comment:
Everything in this file needs to be reviewed by a native Turkish speaker. I want to review and fix the problems when I have some time earlier next year (no pun intended).

Same for `locale/tr/site.json`.

Reply:
/cc @nodejs/nodejs-tr